sdk/keyvault/azure-security-keyvault-secrets/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "java",
+  "TagPrefix": "java/keyvault/azure-security-keyvault-secrets",
+  "Tag": "java/keyvault/azure-security-keyvault-secrets_66f694112f"
```

Comment:
Did we generate this tag procedurally? Will this need to ever be updated in the future?

Reply:
yes, we would need to update this tag when we want to update the recordings on the assets repo.
`test-proxy push`  command is used to push these recordings to the azure-sdk-assets repo.
More details here - 
https://github.com/Azure/azure-sdk-for-java/wiki/Test-Proxy-Migration#running-tests-in-record-mode